playbooks/common/openshift-cluster/metrics/uninstall_metrics.yml

```diff
@@ -0,0 +1,21 @@
+---
+- hosts: oo_first_master
+  tasks:
+  - name: "Remove metrics url from  master config"
+    lineinfile: "state=absent dest=/etc/origin/master/master-config.yaml regexp='^\ \ metricsPublicURL'"
+
+  - name: "Delete metrics objects"
+    command: "{{item}}"
+    with_items:
+      - oc delete all --selector=metrics-infra --namespace openshift-infra --ignore-not-found
+      - oc delete templates --selector=metrics-infra --namespace openshift-infra --ignore-not-found
+      - oc delete sa metrics-deployer --namespace openshift-infra --ignore-not-found
+      - oc delete sa hawkular --namespace openshift-infra --ignore-not-found
+      - oc delete sa heapster --namespace openshift-infra --ignore-not-found
+      - oc delete sa cassandra --namespace openshift-infra --ignore-not-found
+      - oc delete secret metrics-deployer --namespace openshift-infra --ignore-not-found
+      - oc delete secret hawkular-cassandra-certificate --namespace openshift-infra --ignore-not-found
+      - oc delete secret hawkular-cassandra-secrets --namespace openshift-infra --ignore-not-found
+      - oc delete secret hawkular-metrics-account --namespace openshift-infra --ignore-not-found
+      - oc delete secret hawkular-metrics-certificate --namespace openshift-infra --ignore-not-found
+      - oc delete secret hawkular-metrics-secrets --namespace openshift-infra --ignore-not-found
```

Comment:
I'm not familiar with these metrics resources, isn't there a label to select them all? If so, I'd suggest a single call to `oc delete`.

```
oc -n openshift-infra delete --ignore-not-found all,templates,sa,secret -l metrics-infra
```